interface/forms/soap/templates/general_new.html

```diff
@@ -1,52 +1,58 @@
+                            <legend class="">{xl t='Subjective'|escape:'html'}</legend>
+                                <div class="form-group" >
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="subjective" class="form-control" cols="40" rows="6">{$data->get_subjective()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Objective'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="objective" class="form-control" cols="60" rows="6">{$data->get_objective()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Assessment'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="assessment" class="form-control" cols="60" rows="6">{$data->get_assessment()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Plan'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="plan" class="form-control" cols="60" rows="6">{$data->get_plan()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <?php //can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets ?>
+                    <div class="form-group clearfix">
+                        <div class="col-sm-12 text-left position-override">
+                            <div class="btn-group btn-group-pinch" role="group">
                                 <button type="submit" class="btn btn-default btn-save" name="Submit">{xl t='Save'|escape:'html'}</button>
-                                <a href="{$DONT_SAVE_LINK}" class="btn btn-link btn-cancel" onclick="top.restoreSession()">{xl t='Cancel'|escape:'html'}</a>
+                                <button type="button" class="btn btn-link btn-cancel btn-separate-left"onclick="top.restoreSession(); location.href='{$DONT_SAVE_LINK}';">{xl t='Cancel'|escape:'html'}</button>
```

Comment:
hi @zbig01 , understood. Definitely leave it here and then the theme styles will have the option of using this or not. and prob would then be good to rename it to something like optional-btn-separate-left.